src/models/SCLedger.ts

```diff
@@ -0,0 +1,6 @@
+export default interface Ledger {
+  [original: string]: {
+    original: string
```

Comment:
We were able to get rid of it in `SCE`, maybe we can do it here too?

Reply:
Are yor sure?) https://github.com/BigWhaleLabs/seal-cred-echo-frontend/blob/main/src/models/Ledger.ts